docs/cimpl6LanguageReference.md

```diff
@@ -1057,124 +1044,127 @@ With FHIR as the target output, the right-hand side of the map statement can poi
+#### Discriminator Path: `slice on`
 
-##### Slice On
-<!-- MK:  the second sentence needs work, but I don't know what to change it to.  Also - the example (based on the 2nd part = coding) seems to be referencing a FHIR data type, and if the slice is on a FHIR property (includes) - I'm missing the connection. Suggest an example that doesn't have the same word repeating.  Need help on this paragraph and example in #### Slice Strategy   New topic -  The 5# headings for slice on and discriminator type don't stand out in the rendered view  -->
-
-Declare a discriminator path to perform slicing, using `slice on` followed by the discriminator path. This is the path to the `Property` whose data identifies the `slice on` instance belongs to. For example, if `slice on =` _code.coding.code_, the code in each slice must be a unique, fixed value, and every instance must have a code that matches one of the defined slices.
+FHIR requires discriminators to uniquely identify the slice to which an instance of data belongs. The path to the discriminator is specified in the `slice on` parameter. For example, if `slice on = code.coding.code`, the code in each slice must be a unique, fixed value, and every instance must have a code that matches one of the defined slices.
 
 For more explanation of how discriminators are defined in FHIR, see [discriminators](https://www.hl7.org/fhir/profiling.html#discriminator).
 
-##### Discriminator Type
+#### Discriminator Type: `slice on type`
 
-The discriminator type defines how a slice is differentiated from another slice for the same `Property`. By default, CIMPL uses the FHIR **value** discriminator type. However, a different discriminator type can be declared using `slice on type` followed by one of the following FHIR-defined discriminator types:
+The discriminator type defines how the discriminator is used to differentiate instances from one another. By default, CIMPL uses the FHIR **value** discriminator type. However, a different discriminator type can be declared using `slice on type` followed by one of the following FHIR-defined discriminator types:
 
-| `Slice on` Type | Definition |
+| `slice on type` | Interpretation |
 | ------------- | ---------- |
-| **value**| The slices have different values in the discriminator |
-| **exists** | The slices are differentiated by the presence or absence of the `sliced on`|
-| **pattern** |The slices have different values in the discriminator, as determined by testing them against the applicable **ElementDefinition.pattern[x]**|
-| **type** |The slices are differentiated by data type of the discriminator to a specified **profile**|
-|**profile** |Not recommended. The slices are differentiated by conformance of the discriminator to a specified **profile**. This can only be used when the discriminator is an `Entry`.|
+| `value`| The slices have different values in the discriminator |
```

Comment:
Using the FHIR discriminator types (as described on line 1116) the first column s/b bolded.

Reply:
22 instances of the word discriminator -- all bolded (not sure I like this, but that's how it is now)